src/test/resources/pages/instructorFeedbackResultsPageOpen.html

```diff
@@ -199,11 +199,11 @@
           <span class="help-block" style="display:inline;">
             Tips: After selecting the table,
             <kbd>
-              Ctrl + C
+              Ctrl + C or Command + C
```

Comment:
Apologies. I glanced the issue and jumped to the conclusion that this file is related to just macOS when clearly it's general instruction for all platforms.